src/nrf_802154_const.h

```diff
@@ -88,6 +88,16 @@
 #define KEY_ID_MODE_2                0x10                                         ///< Bits containing the 0x10 Key Identifier Mode.
 #define KEY_ID_MODE_3                0x18                                         ///< Bits containing the 0x11 Key Identifier Mode.
 
+#define MAC_CMD_ASSOC_REQ            0x01                                         ///< MAC Association request command frame identifier.
```

Comment:
```suggestion
#define MAC_CMD_ASSOC_REQ            0x01                                         ///< Command frame identifier for MAC Association request.
```
This is quite a long concatenation of nouns... so please rework all the defines in lines 91-99 as this one. Alternative is `Command frame identifier: MAC Association request.`